polkadot/runtime/westend/src/lib.rs

```diff
@@ -1491,7 +1510,12 @@ construct_runtime! {
 		ParaSessionInfo: parachains_session_info::{Pallet, Storage} = 52,
 		ParasDisputes: parachains_disputes::{Pallet, Call, Storage, Event<T>} = 53,
 		ParasSlashing: parachains_slashing::{Pallet, Call, Storage, ValidateUnsigned} = 54,
-		ParaAssignmentProvider: parachains_assigner_parachains::{Pallet, Storage} = 55,
+		// TODO: I just swapped out the pallet here (keeping the same number). I assume this is
+		// fine as we are not exposing any calls, events nor origins, but I am not 100% sure.
+		ParaAssignmentProvider: parachains_assigner_v1::{Pallet, Storage} = 55,
```

Comment:
I guess it was done to match [rococo](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/rococo/src/lib.rs#L1407), but I'm finding it confusing. To my understanding, `parachains_assigner_parachains` is the legacy assigner, so it should indeed come first and then it can stay pallet `55`. Also, some simple comments here indicating how all these assigners are different, would be very helpful. Because `ParaAssignment` and `ParachainAssignment` look very similar.